app/experimenter/experiments/email.py

```diff
@@ -22,3 +23,46 @@ def send_review_email(experiment_name, experiment_url, needs_attention):
         [settings.EMAIL_REVIEW],
         fail_silently=False,
     )
+
+
+def send_intent_to_ship_email(experiment_id):
+
+    def make_url(uri):
+        return urljoin("https://{}".format(settings.HOSTNAME), uri)
+
+    experiment = Experiment.objects.prefetch_related(
+        "locales", "countries"
```

Comment:
We should move this into https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/models.py#L20 so that you don't have to explicitly do it.